checkton-local.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
helps if user is using a different default shell that doesn't support `mapfile` (like in my case, on MacOS)
```suggestion
#!/usr/bin/env bash
```